### PR TITLE
[Enhance]Unittest reconstruction

### PR DESCRIPTION
**Pull request by xiliu8006**
Modify the directory structure of unit test

### CODE REVIEW

File: tests/test_models/test_detectors.py [hunk 1]
```diff
@@ -21,11 +21,11 @@ def _get_config_directory():
     """Find the predefined detector config directory."""
     try:
         # Assume we are running in the source mmdetection repo
```
Comment (Tai-Wang) [inline tests/test_models/test_detectors.py:23]: Assume we are running in the source mmdetection3d repo

File: tests/test_models/test_forward.py [hunk 2]
```diff
@@ -14,11 +14,11 @@ def _get_config_directory():
     """Find the predefined detector config directory."""
     try:
         # Assume we are running in the source mmdetection repo
```
Comment (Tai-Wang) [inline tests/test_models/test_forward.py:16]: Assume we are running in the source mmdetection3d repo

File: tests/test_models/test_heads/test_heads.py [hunk 3]
```diff
@@ -23,11 +23,11 @@ def _get_config_directory():
     """Find the predefined detector config directory."""
     try:
         # Assume we are running in the source mmdetection repo
```
Comment (Tai-Wang) [inline tests/test_models/test_heads/test_heads.py:25]: Modify others correspondingly

File: tests/test_runtime/test_apis.py [hunk 4]
```diff
@@ -12,11 +12,11 @@ def _get_config_directory():
     """Find the predefined detector config directory."""
     try:
         # Assume we are running in the source mmdetection repo
```
Comment (Tai-Wang) [inline tests/test_runtime/test_apis.py:14]: mmdetection3d

File: tests/test_runtime/test_config.py [hunk 5]
```diff
@@ -5,11 +5,11 @@ def _get_config_directory():
     """Find the predefined detector config directory."""
     try:
         # Assume we are running in the source mmdetection repo
```
Comment (Tai-Wang) [inline tests/test_runtime/test_config.py:7]: mmdetection3d

**Review by Tai-Wang — COMMENTED** [review]
test_indoor_eval and test_kitti_eval should be moved to test_metrics?

**Review by Tai-Wang — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**